notebooks/A_CNN_Annotation_with_Napari.ipynb

```diff
@@ -204,32 +205,34 @@
     "def get_image_patch(layers, coords, shape=64):\n",
     "    \"\"\" Get an image patch from the image layer data. \"\"\"\n",
     "    \n",
-    "    frame, y_coo, x_coo = [int(coo) for coo in coords]\n",
-    "    pad = shape // 2\n",
-    "    \n",
-    "    # Read the image from layers:\n",
-    "    # TODO: soft-code ## presuming this has already been done? nd.\n",
-    "    image = layers[0].data[frame]\n",
-    "    im_h, im_w = image.shape\n",
-    "    edge_dist = [y_coo, im_h - y_coo, x_coo, im_w - x_coo]\n",
-    "    padded = \"\"\n",
-    "    \n",
-    "    # Check if padding is needed:\n",
-    "    if any([item < pad for item in edge_dist]):\n",
-    "        pad_width = int(pad - min(edge_dist))\n",
-    "        image = np.pad(array=image, pad_width=pad_width, mode='constant')\n",
-    "        y_coo += pad_width\n",
-    "        x_coo += pad_width\n",
-    "        padded = \"_flagged\"\n",
-    "    \n",
-    "    # Crop patch around the point:\n",
-    "    patch = np.array([row[x_coo - pad : x_coo + pad] for row in image[y_coo - pad : y_coo + pad]]).astype(np.uint8)\n",
+    "    patches = [[], False]\n",
```

Comment:
I don't really like this patches list, why not have two variables `patches` and `flagged` so you can return the tuple